stock_partner_lot/models/res_partner.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+from openerp import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    quant_ids = fields.One2many('stock.quant', 'owner_id', string='Owned products')
+    quant_count = fields.Integer('Owned Products', compute='_quant_count')
+
+    @api.multi
+    def _quant_count(self):
+        for partner in self:
+            try:
```

Comment:
It used to be that if you lacked access to the related model, you would get an access error, seems it's no longer an issue at least in this case.